gspread/auth.py

```diff
@@ -138,14 +130,24 @@ def oauth(scopes=DEFAULT_SCOPES, flow=local_server_flow):
         sh = gc.open("A spreadsheet")
         sh.sheet1.update('A1', '42')   # <-- this will not work
 
+    If you're storing your user credentials in a place other than the
+    default, you may provide a path to that file like so::
+
+        gc = gspread.oauth(credentials_filename='/alternative/path/authorized_user.json')
+
 
     :param list scopes: The scopes used to obtain authorization.
     :param function flow: OAuth flow to use for authentication.
         Defaults to :meth:`~gspread.auth.local_server_flow`
+    :param str credentials_filename: Filepath (including name) pointing to
+        a user's service account `.json` file.
```

Comment:
This could lead to users's confusion: although the files are similar, *service account* is not  the authorized user json file. 
See: [google.oauth2.credentials.Credentials.from_authorized_user_file](https://google-auth.readthedocs.io/en/stable/reference/google.oauth2.credentials.html#google.oauth2.credentials.Credentials.from_authorized_user_file)

Reply:
I rephrased the docstring a bit. I hope that's what you meant?